test/tslint-custom-rules-with-dir.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
can we move this into a new dir `test/config` please? I'd like to not clutter top-level directories.